src/routes.js

```diff
@@ -77,6 +76,17 @@ export default (
           UserRole.Owner,
           UserRole.Mentor,
         ]}
+        routePermissions={[RoutePermissions.weeklySummariesReport, RoutePermissions.reports]}
+      />
+      <ProtectedRoute
+        path="/reports"
+        exact
+        component={Reports}
+        allowedRoles={[
```

Comment:
Could you explain this part to me? Does it mean only admins and owners have permission to Reports? But the " see all the reports" tab is not added for owners as I seen.